test/puppetlabs/trapperkeeper/services/metrics/metrics_service_test.clj

```diff
@@ -21,6 +24,15 @@
   ([resp keywordize?]
    (-> resp :body slurp (json/parse-string keywordize?))))
 
+(defn jolokia-encode
```

Comment:
@rlinehan, @Sharpie - did either of you file a JIRA ticket to track the docs work for the '/metrics/v2' endpoint?  I didn't yet but can do if you haven't before I forget about this again.